_posts/2022-04-21-tardigrade-launch.md

```diff
@@ -0,0 +1,323 @@
+engineering to get us to the current status. The latest Trino release includes
+all the work from Project Tardigrade. Read on to learn how it all works and how
+to enable the fault tolerant query processing in Trino.
+
+<!--more-->
+
+## What is Project Tardigrade?
+
+What we love most about Trino is that you get fast query speeds, and you can
+iterate fast with intuitive error messages, interactive experience, and query
+federation.
+
+One of the big problems that persisted a long time is the configuring, tuning,
+and managing Trino for long-running ETL workloads is very difficult. Following
+are just some of the problems you have to deal with:
+
+* _Reliable landing times:_ Queries that run for hours can fail. Restarting
+  them from scratch wastes resources and makes it hard for you to meet
+  your data landing SLA.
+* _Cost-efficient clusters:_ Trino queries that need terabytes of distributed
+  memory require extremely large clusters due to the lack of iterative
+  execution.
+* _Concurrency:_ Multiple independent clients may submit their queries
+  concurrently. Due to the lack of available resources at a certain moment some
+  of these queries may need to be killed and restarted from zero after a
+  while. This makes the landing time hardly predictable.
+
+[Structuring your workload](https://engineering.salesforce.com/how-to-etl-at-petabyte-scale-with-trino-5fe8ac134e36)
+to avoid these problems can be done by a team of experts. But that is not
+accessible to most Trino users.
+
+The goal of Project Tardigrade is to provide an "out of the box" solution for the
+problems mentioned above. We’ve designed a new
+[fault tolerant execution architecture](https://github.com/trinodb/trino/wiki/Fault-Tolerant-Execution)
+that allows us to implement an advanced resource-aware scheduling with granular
+retries.
+
+What this means is:
+
+* When your long-running queries experience a failure, they don’t have to start
+  from scratch.
+* When queries require more memory than currently available in the cluster
+  they are still able to succeed.
+* When multiple queries are submitted concurrently they they are able to share
+  resources in a fair way, and make steady progress.
+
+Trino does all the hard work of allocating, configuring and maintaining query
+processing behind the scenes. Instead of spending time tuning Trino clusters to
+match your workload requirements, or reorganizing your workload to match your
+Trino cluster capabilities, you can spend your time on analytics and delivering
+business value. And most importantly, your heart won’t skip a beat when you
+wake up in the morning wondering whether that query landed on time.
+
+## What did we test so far?
+
+Since there's no publicly available testing query set for ETL use cases, we
+handcrafted more than a hundred ETL-like queries based on the
+[TPC-H](https://github.com/trino/trino-verifier-queries/tree/main/src/main/resources/queries/tpch/etl)
```

Comment:
This link and the TPC-DS links are dead for me. Is this for a private repo? Also the url should be based with github.com/trinodb not github.com/trino

Reply:
Fixed